datasetup/build_nonUS_setup.R

```diff
@@ -7,20 +7,19 @@
 # ## Configuration Items
 #
 # ```yaml
-# data_path: <path to directory>
 # subpop_setup:
 #   modeled_states: <list of country ISO3 codes> e.g. ZMB, BGD, CAN
-#   mobility: <path to file relative to data_path> optional; default is 'mobility.csv'
-#   geodata: <path to file relative to data_path> optional; default is 'geodata.csv'
+#   mobility: <path to file> optional; default is 'mobility.csv'
```

Comment:
yeah, this seems fine as well, though a little less explicit - probably just use this language for `..._US_...` as well?